### ISSUE TITLE
Can't override VM options

### ISSUE THREAD

**Issue by fkrull** [flathub/com.jetbrains.IntelliJ-IDEA-Ultimate#21]
The Flatpak currently [hardcodes the IDEA_VM_OPTIONS environment variable](https://github.com/flathub/com.jetbrains.IntelliJ-IDEA-Ultimate/blob/31e3f52ba06730801f3b0876413940d7bede7c39/com.jetbrains.IntelliJ-IDEA-Ultimate.json#L12) to the options file shipped with IntelliJ. This makes it impossible to set custom VM options in `~/.IntelliJIdea2019.1/config/idea64.vmoptions` because the file is apparently never read. This is easily verifiable by checking the command line of the main java process.

This should be changed because as of now it's not possible to increase e.g. the JVM heap size for IntelliJ without resorting to `flatpak override`.

*Workaround*: `flatpak override --env=IDEA_VM_OPTIONS= com.jetbrains.IntelliJ-IDEA-Ultimate` restores the default behaviour, as far as I can tell.

**Comment by x80486** [comment]
Indeed...removing the line `--env=IDEA_VM_OPTIONS...` from the manifest will do it.

I'll change it to initially setup the defaults and use `${HOME}/.IntelliJIdea{VERSION}/config/idea64.vmoptions` :sunglasses:
